docs/testplans/hatchery/hatchery-executable-test-plan.md~

```diff
@@ -0,0 +1,38 @@
+# Object Management API (aka: Shepherd) test plan
+
```

Comment:
This is great - thanks!
I think you don't want to checkin this file - maybe add a line to `.gitignore` ...

Reply:
ugh, I totally missed this message.
Let me fix that...